client/my-sites/stats/stats-card-upsell/stats-upsell-copy.tsx

```diff
@@ -0,0 +1,197 @@
+import { localizeUrl } from '@automattic/i18n-utils';
+import { translate } from 'i18n-calypso';
+import {
+	SUPPORT_URL,
+	INSIGHTS_SUPPORT_URL,
+	JETPACK_SUPPORT_URL_TRAFFIC,
+	JETPACK_SUPPORT_VIDEOPRESS_URL_STATS,
+} from 'calypso/my-sites/stats/const';
+import {
+	STATS_FEATURE_DATE_CONTROL,
+	STATS_FEATURE_UTM_STATS,
+	STATS_TYPE_DEVICE_STATS,
+	STAT_TYPE_CLICKS,
+	STAT_TYPE_COUNTRY_VIEWS,
+	STAT_TYPE_FILE_DOWNLOADS,
+	STAT_TYPE_REFERRERS,
+	STAT_TYPE_SEARCH_TERMS,
+	STAT_TYPE_TOP_AUTHORS,
+	STAT_TYPE_TOP_POSTS,
+	STAT_TYPE_VIDEO_PLAYS,
+	STAT_TYPE_EMAILS_SUMMARY,
+	STAT_TYPE_INSIGHTS_ALL_TIME_STATS,
+	STAT_TYPE_INSIGHTS_MOST_POPULAR_TIME,
+	STAT_TYPE_INSIGHTS_MOST_POPULAR_DAY,
+	STAT_TYPE_INSIGHTS_ALL_TIME_INSIGHTS,
+	STAT_TYPE_TAGS,
+	STAT_TYPE_COMMENTS,
+} from '../constants';
+
+const jetpackTrafficSupportLinkWithAnchor = ( anchor: string ) => {
+	return (
+		<a
+			href={ localizeUrl( `${ JETPACK_SUPPORT_URL_TRAFFIC }#${ anchor }` ) }
+			target="_blank"
+			rel="noopenner noreferrer"
+		/>
+	);
+};
+
+const trafficSupportLinkWithAnchor = ( anchor: string ) => {
+	return (
+		<a
+			href={ localizeUrl( `${ SUPPORT_URL }#${ anchor }` ) }
+			target="_blank"
+			rel="noopenner noreferrer"
+		/>
+	);
+};
+
+const insightsSupportLinkWithAnchor = ( anchor: string ) => {
+	return (
+		<a
+			href={ localizeUrl( `${ INSIGHTS_SUPPORT_URL }#${ anchor }` ) }
+			target="_blank"
+			rel="noopenner noreferrer"
+		/>
+	);
+};
+
+const getUpsellCopy = ( statType: string ) => {
+	switch ( statType ) {
+		case STAT_TYPE_CLICKS:
+			return translate(
+				'Learn what {{link}}external links{{/link}} your visitors click on your site to reveal their areas of interest.',
+				{
+					components: {
+						link: jetpackTrafficSupportLinkWithAnchor( 'clicks' ),
```

Comment:
I think we should create a component to determine whether it is a Jetpack site or not and then render the appropriate link (jetpack.com or wordpress.com). In this case, the Clicks upsell will always point to Jetpack, regardless of whether it is a Jetpack site or a WordPress.com site.

Reply:
This was addressed here: p1734545817763079-slack-C068X5XJT3J
It is okay to keep using Jetpack.com support docs as you proposed :)